servers/arxiv-mcp-server/server.yaml

```diff
@@ -0,0 +1,49 @@
+name: arxiv-mcp-server
+image: mcp/arxiv-mcp-server
+type: server
+meta:
+  category: search
+  tags:
+    - arxiv
+    - research
+    - academic
+    - papers
+    - search
+    - ai
+about:
+  title: ArXiv MCP Server
+  description: |
+    The ArXiv MCP Server provides a comprehensive bridge between AI assistants and arXiv's research repository through the Model Context Protocol (MCP). 
+    
+    Features:
+    • Search arXiv papers with advanced filtering
+    • Download and store papers locally as markdown
+    • Read and analyze paper content
+    • Deep research analysis prompts
+    • Local paper management and storage
+    
+    Perfect for researchers, academics, and AI assistants conducting literature reviews and research analysis.
+  icon: https://avatars.githubusercontent.com/u/15390319?v=4
+source:
+  project: https://github.com/jasonleinart/arxiv-mcp-server
+run:
+  volumes:
+    - '{{arxiv-mcp-server.storage_path}}:/app/papers'
+config:
+  description: Configure local storage path for downloaded papers
+  env:
+    - name: STORAGE_PATH
+      example: /Users/local-test/papers
+      value: '{{arxiv-mcp-server.storage_path}}'
```

Comment:
@jasonleinart Looks like you can remove this one now, since you added `ARXIV_STORAGE_PATH`